FinSearchUnified/Tests/Subscriber/FrontendTest.php

```diff
@@ -79,6 +79,68 @@ public function vendorProvider()
         ];
     }
 
+    public function legacyUrlProvider()
+    {
+        return [
+            'Normal search query' => [
```

Comment:
Try to avoid words like "normal" since that could mean anything. Maybe `Query without filters`?